src/Glpi/Helpdesk/Tile/TilesManager.php

```diff
@@ -114,7 +114,14 @@ public function getTilesForItem(
             }
 
             // Try to load tile from database
-            if (!$tile->getFromDb($row['items_id_tile'])) {
+            try {
+                if (!$tile->getFromDb($row['items_id_tile'])) {
+                    continue;
+                }
+            } catch (InvalidTileException $e) {
+                // Should not happen unless the database is manually edited
+                // Log the error but do not block the exectuion.
+                trigger_error("Unable to load linked form", E_USER_WARNING);
```

Comment:
To get the full stack trace in the logs.

```suggestion
                /** @var \Psr\Log\LoggerInterface $PHPLOGGER */
                global $PHPLOGGER;
                $PHPLOGGER->error("Unable to load linked form", ['exception' => $e]);
```